### ISSUE TITLE
Question about F.log_softmax(x, dim=0) in the model class

### ISSUE THREAD

**Issue by shirui-japina** [yajirobe3/wine_classifier_DNN#1]
恐らくですが、モデルの最後の箇所の`F.log_softmax(x, dim=0)`は間違いだと思います。
理由ですが、使用する損失函数`criterion = nn.CrossEntropyLoss()`は

> This criterion combines nn.LogSoftmax() and nn.NLLLoss() in one single class.

と記述されています、[原文はこちら](https://pytorch.org/docs/stable/nn.html#torch.nn.CrossEntropyLoss)。
つまり損失函数値を求める過程で、自動にsoftmax されます。
実際に以下コードでも動作し、正解率は90％代でした。理由は現状不明です。
```
    def forward(self, x):
        x = F.relu(self.fc1(x)) # ReLU: max(x, 0)
        x = F.relu(self.fc2(x))
        x = F.relu(self.fc3(x))
        x = F.relu(self.fc4(x))
        x = F.relu(self.fc5(x))
        x = self.fc6(x)
        # return F.log_softmax(x, dim=0)
        return x
```
